lib/actions/apiV2.js

```diff
@@ -94,16 +94,16 @@ function createGraphQLQueryAction(
   errorAction,
   options
 ) {
-  const endpoint = 'index/graphql'
   const fetchOptions = {
     body: JSON.stringify({ query, variables }),
     headers: { 'Content-Type': 'application/json' },
     method: 'POST'
   }
-  return createQueryAction(endpoint, responseAction, errorAction, {
+  return createQueryAction(null, responseAction, errorAction, {
     ...options,
     fetchOptions,
-    noThrottle: true
+    noThrottle: true,
+    url: '/otp/gtfs/v1'
```

Comment:
Can `url` be transfered to the `endpoint` argument?

Reply:
`endpoint` appends `routers/default`